src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs

```diff
@@ -19,6 +21,7 @@ public class NpgsqlNetTopologySuiteMethodCallTranslatorPlugin : IMethodCallTrans
             => Translators = new IMethodCallTranslator[]
             {
                 new NpgsqlGeometryMethodTranslator(sqlExpressionFactory, typeMappingSource),
+                new NpgsqlGeometryFunctionTranslator(sqlExpressionFactory),
```

Comment:
Any reason to have a separate translator just for Transform? I understand it's an extension on DbFunctions and not a native NetTopologySuite method, but I'd rather we just put them in the same place.